docs/automation-rules.rst

```diff
@@ -99,6 +108,13 @@ Activate only new branches that belong to the ``1.x`` release
 - Version type: ``Branch``
 - Action: ``Activate version``
 
+Automatically delete an active version of a deleted branch
```

Comment:
```suggestion
Delete an active version when a branch is deleted
```

There is no need to mention "Automatically". Also, I re-wrote the sentence. I think it's clearer.